hw/hal/syscfg.yml

```diff
@@ -42,7 +42,11 @@ syscfg.defs:
             If set to 0 software breakpoints placed with HAL_DEBUG_BREAK macro will not
             be executed.
         value: 1
-
+    HAL_SBRK:
+        description: >
+            If set HAL provides standard implementation of _sbrk function.
+            It also provides _sbrkInit function that sets up heap space for malloc.
+        value: 0
```

Comment:
Wouldn't it be better to make this default to `1` and leave the few HALs that must provide their own implementation reset it?

Reply:
done